src/Google/AdsApi/Common/AdsSoapClient.php

```diff
@@ -74,8 +74,10 @@ public function __construct($wsdl, array $options = null)
 
         $localWsdlPath = self::getLocalWsdlPath($wsdl);
         if (!file_exists($localWsdlPath)) {
-            trigger_error("Local WSDLs bundled with this library were not found.\n",
-                E_USER_NOTICE);
+            trigger_error(
```

Comment:
Was this format correction done by CodeSniffer? Or is it a manual edit?

Reply:
This was an issue flagged by CodeSniffer than i manually fixed.

For reference, this is the output I received:

```
$ vendor/bin/phpcs src --standard=phpcs_ruleset.xml -np
............................................................  60 / 155 (39%)
............................................................ 120 / 155 (77%)
..E................................                          155 / 155 (100%)



FILE: /home/vagrant/code/googleads-php-lib/src/Google/AdsApi/Common/AdsSoapClient.php
---------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------
 77 | ERROR | [x] Opening parenthesis of a multi-line function call must be the last content on the line
---------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
---------------------------------------------------------------------------------------------------------

Time: 9.61 secs; Memory: 10MB
```

Would you like me to revert this?

Reply:
Please keep it. I am asking this question because when I run CodeSniffer, it doesn't show the same error.